alma/controllers/admin/AdminAlmaRefunds.php

```diff
@@ -30,8 +30,8 @@
 
 // TODO: Autoloader Trait Automatically
 if (version_compare(_PS_VERSION_, '1.7', '<')) {
-    require_once(_PS_MODULE_DIR_ . 'alma/lib/Utils/AjaxTrait.php');
-    require_once(_PS_MODULE_DIR_ . 'alma/lib/Utils/OrderDataTrait.php');
+    require_once _PS_MODULE_DIR_ . 'alma/lib/Utils/AjaxTrait.php';
```

Comment:
you should not do that because of the autoload refactoring in progress in an other branch.
this modification will produce a merge or rebase conflict.

I am not sure to use `require_once` without `()` is more clear or clean 🧐

Reply:
Exactly. I'm fallback for no create conflit. 

It's a recommandation of phpcs of no use `()` in `require_once`